### PR TITLE
rpc: update docs

### PR DESCRIPTION
**Pull request by samricotta**
Closes issue [#9522](https://github.com/tendermint/tendermint/issues/9522) by adding genesis chunked to the rpc docs

### CODE REVIEW

File: rpc/openapi/openapi.yaml [hunk 1]
```diff
@@ -911,6 +911,30 @@ paths:
...
+
+        Upon success, the `Cache-Control` header will be set with the default
+        maximum age.
+      responses:
+        "200":
+          description: Genesis chunked results.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GenesisChunkedResponse"
```
Comment (cmwaters) [inline rpc/openapi/openapi.yaml:931]: Does this component already exist somewhere?

File: rpc/openapi/openapi.yaml [hunk 2]
```diff
@@ -911,6 +911,30 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
+  /genesis_chunked:
+     get:
+      summary: Get Genesis Chunked
```
Comment (cmwaters) [inline rpc/openapi/openapi.yaml:916]:
```suggestion
      summary: Get Genesis in multiple chunks
```

File: rpc/openapi/openapi.yaml (hunk 1 above)
Comment (thanethomson) [reply rpc/openapi/openapi.yaml:931]: No, this schema still needs to be added to the bottom of the doc.

Comment (samricotta) [reply rpc/openapi/openapi.yaml]: Yes, just added this one in

File: rpc/openapi/openapi.yaml [hunk 3]
```diff
@@ -911,6 +911,39 @@ paths:
...
+        - Info
+      description: |
+        Get genesis document in multiple chunks to make it
+        easier to iterate through larger gensis structures.
+
+        Upon success, the `Cache-Control` header will be set with the default
+        maximum age.
+      parameters:
+        - in: query
+          name: chunkID
```
Comment (thanethomson) [inline rpc/openapi/openapi.yaml:928]: Looking at https://github.com/tendermint/tendermint/blob/99a7ac84dca30676fd544be18c6df2880a14429f/rpc/core/routes.go#L22 shouldn't this be:

```suggestion
          name: chunk
```
?

**Comment by jmalicevic** [comment]
Looking at the way it was in the master, looks good to me. Is there a way to test it ,to make sure there is nothing breaking due to an extra space or so?

**Comment by jmalicevic** [comment]
Is there a way to confirm that the rpc docs display this correctly before merging? I know .yml files can be sensitive to spaces etc. Otherwise, looks good to me.

**Comment by thanethomson** [comment]
> Is there a way to confirm that the rpc docs display this correctly before merging? I know .yml files can be sensitive to spaces etc. Otherwise, looks good to me.

The easiest way to do this from `main` is to run:

```bash
make serve-docs
```

from the root of the repo. This should serve the docs site from `http://0.0.0.0:8080` on your local machine. If you get an exception when running the above, try running:

```bash
NODE_OPTIONS=--openssl-legacy-provider make serve-docs
```

**Review by cmwaters — APPROVED** [review]
LGTM

**Comment by cmwaters** [comment]
Is our openapi stuff versioned? i.e. does this need to be backported to the versions that the endpoint is available in?

**Review by thanethomson — APPROVED** [review]

**Pull request merged**